emerlin2caom2/main_app.py

```diff
@@ -60,18 +97,24 @@ def fits_plane_metadata(self, observation, fits_full_name, images):
         :param images: String name of fits file, no path
         :returns: plane created for fits file to be passed to the artifact function
         '''
-        # plane_id_full = storage_name + '/weblog/images/' + directory + '/'
+
         plane = Plane(images)
         observation.planes[images] = plane
         fits_header_data = fr.header_extraction(fits_full_name + images)
 
         position = Position()
         plane.position = position
-        plane.position.shape = Point(fits_header_data['ra_deg'], fits_header_data['dec_deg'])
+        centre = Point(fits_header_data['ra_deg'], fits_header_data['dec_deg'])
+        width = abs(fits_header_data['pix_width'] * fits_header_data['pix_width_scale'])
+        height = abs(fits_header_data['pix_length'] * fits_header_data['pix_length_scale'])
+        radius = 0.5 * width
+        # plane.position.bounds = shape.Box(centre, radius)
+        plane.position.bounds = shape.Circle(centre, radius)
+        # should be box but is unsupported by the writer, neither is point
 
         energy = Energy()
         plane.energy = energy
-        plane.energy.restwav = fits_header_data['central_freq']  # change freq to wav and check against model
+        plane.energy.restwav = 3e8/fits_header_data['central_freq']  # change freq to wav and check against model
```

Comment:
The ivoa standard I believe requests specific conversion value of 2.99792458x10^8, so not sure if you want to update this? 
This conversion may be redundant code as there is a mini-function doing this in casa_reader.py. Do we want to move the casa_reader all to in-line conversions like the new one? Or consolidate somehow?
Also, the PR notes say that this branch includes adding conversion of wavelength to frequency, but caom accepts wavelength as a standard (not frequency), and I cannot find any places in this code that do convert wavelength to frequency.

Reply:
I must have missed that function, I have added this change. 
As for the wavelength to frequency part, I think I just got the order wrong in the PR.